package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@cyclimse, Keep in mind that no external packager nor a package.json will be uploaded to the serverless platform. ESbuild will bundle everything needed into one JS file, and a sourcemap, those will be uploaded.

Reply:
You shouls change  keepOutputDirectory: to true, and then run serverless package. This will let you inspect the build directory.

Reply:
Sorry, I wasn't clear at all. This file was committed to the top level of the serverless-examples git repo. I don't think it's needed to have it there and it might become a problem. I would much prefer to keep all examples resources in their own folders.

Reply:
I think that was in error. It slipped by. Only the files in the example directory are important.

Reply:
No worries it happens all the time. If you have some time, I would really appreciate if you could clean them up.